### PR TITLE
Jaeger: Remove browser access mode

### LINKED ISSUE #25699: Jaeger datasource always proxies, event when Access: Browser is selected
**What happened**: Jaeger requests are made through Grafana proxy mode, even when the datasource is configured to make requests directly from the Browser:

browser-mode
request

Appears to be the result of: https://github.com/grafana/grafana/blob/5211a23f8d921714d839898a8e503ef8987d0773/public/app/plugins/datasource/jaeger/datasource.ts#L92
**What you expected to happen**: Requests were made to the backend from the Browser directly

**How to reproduce it (as minimally and precisely as possible)**: Configure a Jaeger datasource with Access: Browser. Open Explore mode and see that requests are made to the proxy.

**Anything else we need to know?**: If the Grafana backend does not itself have access to the datasource URL then this prevents use of the Jaeger datasource plugin.

**Environment**:
- Grafana version: 7.0.3
- Data source type & version: Jaeger
- OS Grafana is installed on: Ubuntu 16.04
- User OS & Browser: macOS 10.15.5, Safari 13.1.1
- Grafana plugins:
- Others:

### PR DESCRIPTION
**Pull request by zoltanbedi**
**Which issue(s) this PR fixes**:

Fixes #25699

### CODE REVIEW

File: docs/sources/datasources/jaeger.md [hunk 1]
```diff
@@ -15,15 +15,14 @@ Just add it as a data source and you are ready to query your traces in [Explore]
...
-| `URL`        | The URL of the Jaeger instance, for example, `http://localhost:16686`                                                                 |
-| `Access`     | Server (default) = URL needs to be accessible from the Grafana backend/server. Browser = URL needs to be accessible from the browser. |
-| `Basic Auth` | Enable basic authentication to the Jaeger data source.                                                                                |
-| `User`       | User name for basic authentication.                                                                                                   |
-| `Password`   | Password for basic authentication.                                                                                                    |
+| Name         | Description                                                                                     |
+| ------------ | ----------------------------------------------------------------------------------------------- |
+| `Name`       | The data source name. This is how you refer to the data source in panels, queries, and Explore. |
```
Comment (osg-grafana) [inline docs/sources/datasources/jaeger.md:20]:
```suggestion
| `Name`       | The data source name in panels, queries, and Explore. |
```

File: docs/sources/datasources/jaeger.md [hunk 2]
```diff
@@ -15,15 +15,14 @@ Just add it as a data source and you are ready to query your traces in [Explore]
...
+| `Default`    | Data source will be pre-selected for new panels.                                                |
```
Comment (osg-grafana) [inline docs/sources/datasources/jaeger.md:21]:
```suggestion
| `Default`    | The pre-selected data source for a new panel.                                                |
```

File: docs/sources/datasources/jaeger.md [hunk 3]
```diff
@@ -15,15 +15,14 @@ Just add it as a data source and you are ready to query your traces in [Explore]
...
+| `URL`        | The URL of the Jaeger instance, for example, `http://localhost:16686`                           |
```
Comment (osg-grafana) [inline docs/sources/datasources/jaeger.md:22]:
```suggestion
| `URL`        | The URL of the Jaeger instance. For example, `http://localhost:16686`.                           |
```

File: docs/sources/datasources/jaeger.md [hunk 4]
```diff
@@ -15,15 +15,14 @@ Just add it as a data source and you are ready to query your traces in [Explore]
...
+| `Basic Auth` | Enable basic authentication to the Jaeger data source.                                          |
```
Comment (osg-grafana) [inline docs/sources/datasources/jaeger.md:23]: "to" or "for" here? How would you say this in Hungarian?

File: docs/sources/datasources/jaeger.md [hunk 5]
```diff
@@ -15,15 +15,14 @@ Just add it as a data source and you are ready to query your traces in [Explore]
...
+| `User`       | User name for basic authentication.                                                             |
```
Comment (osg-grafana) [inline docs/sources/datasources/jaeger.md:24]:
```suggestion
| `User`       | Specify a user name for basic authentication.                                                             |
```

File: docs/sources/datasources/jaeger.md [hunk 6]
```diff
@@ -15,15 +15,14 @@ Just add it as a data source and you are ready to query your traces in [Explore]
...
+| `Password`   | Password for basic authentication.                                                              |
```
Comment (osg-grafana) [inline docs/sources/datasources/jaeger.md:25]:
```suggestion
| `Password`   | Specify a password for basic authentication.                                                              |
```

**Review by osg-grafana — CHANGES_REQUESTED** [review]

File: docs/sources/datasources/jaeger.md (hunk 4 above)
Comment (zoltanbedi) [reply docs/sources/datasources/jaeger.md:23]: I would say for

File: docs/sources/datasources/jaeger.md [hunk 7]
```diff
@@ -17,12 +17,12 @@ To access Jaeger settings, click the **Configuration** (gear) icon, then click *
 
 | Name         | Description                                                                                     |
 | ------------ | ----------------------------------------------------------------------------------------------- |
-| `Name`       | The data source name. This is how you refer to the data source in panels, queries, and Explore. |
-| `Default`    | Data source will be pre-selected for new panels.                                                |
-| `URL`        | The URL of the Jaeger instance, for example, `http://localhost:16686`                           |
+| `Name`       | The data source name in panels, queries, and Explore. |
+| `Default`    | The pre-selected data source for a new panel.                                                |
+| `URL`        | The URL of the Jaeger instance. For example, `http://localhost:16686`.                           |
 | `Basic Auth` | Enable basic authentication to the Jaeger data source.                                          |
```
Comment (osg-grafana) [inline docs/sources/datasources/jaeger.md:23]:
```suggestion
| `Basic Auth` | Enable basic authentication for the Jaeger data source.                                          |
```

**Review by osg-grafana — CHANGES_REQUESTED** [review]

**Review by osg-grafana — APPROVED** [review]
I assume a linter will fix those table widths.

**Review by aocenas — APPROVED** [review]

**Pull request merged**